src/components/SelectionList/ListItemCheckbox.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import {PressableWithFeedback} from '@components/Pressable';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+
+type ListItemCheckboxProps = {
+    accessibilityLabel?: string;
+    isSelected: boolean;
+    isDisabled: boolean;
+    isDisabledCheckbox: boolean;
+    onPress: () => void;
+};
+
+function ListItemCheckbox({accessibilityLabel, isSelected, isDisabled, isDisabledCheckbox, onPress}: ListItemCheckboxProps) {
```

Comment:
There are 2 components in the codebase:
`/components/Checkbox` and `/components/CheckboxWithLabel`

Do you mind taking a quick look to see if perhaps we could use them here? maybe they are not that different. It would be nice to not create yet another checkbox specific only to list